developer-docs-site/static/examples/typescript/dapp-example/src/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+
+// Wait until DOMContentLoaded so that window.aptos is initialized.
+document.addEventListener("DOMContentLoaded", () => {
+  const root = ReactDOM.createRoot(
+    document.getElementById("root") as HTMLElement
+  );
+  root.render(
+    <React.StrictMode>
+      <App />
+    </React.StrictMode>
+  );
+});
```

Comment:
We may still want to have a notification to say that aptos is injected. But I suppose this is good example on how to get it to work for now.

Reply:
I updated it to use `window.onload` instead of `DOMContentLoaded`.